Upstream: yes
Reason: fixes usage with busybox man

From c9f54e7a411613cbfba2b200d35059f5bcb87113 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ville=20Skytt=C3=A4?= <ville.skytta@iki.fi>
Date: Fri, 15 Apr 2016 07:02:33 +0300
Subject: [PATCH] man: Prioritize MANPATH, simplify, add fallback e.g. for
 busybox, fixes #28

---
 completions/man | 15 ++++-----------
 1 file changed, 4 insertions(+), 11 deletions(-)

diff --git a/completions/man b/completions/man
index e1b587d..628e43c 100644
--- a/completions/man
+++ b/completions/man
@@ -56,17 +56,10 @@ _man()
         return 0
     fi
 
-    local manpath
-    if [[ $OSTYPE == *@(darwin|linux|freebsd|cygwin)* ]] || _userland GNU; then
-        manpath=$( manpath 2>/dev/null || command man -w )
-    else
-        manpath=$MANPATH
-    fi
-
-    if [[ -z $manpath ]]; then
-        COMPREPLY=( $( compgen -c -- "$cur" ) )
-        return 0
-    fi
+    local manpath="$MANPATH"
+    [[ -z $manpath ]] && \
+        manpath=$( manpath 2>/dev/null || command man -w 2>/dev/null )
+    [[ -z $manpath ]] && manpath="/usr/share/man:/usr/local/share/man"
 
     # determine manual section to search
     local sect
-- 
2.8.1

